Apromore-Custom-Plugins/Calendar-Portal/src/main/java/org/apromore/plugin/portal/calendar/controllers/Calendars.java

```diff
@@ -390,8 +390,13 @@ public void onClickRestoreBtn() {
         Notification.info(infoText);
     }
 
-    private void updateApplyCalendarButton() {
+    private void updateButtons() {
         applyCalendarBtn.setDisabled(calendarListbox.getSelectedCount() <= 0 || !canEdit);
+        restoreBtn.setDisabled(
+            !canEdit ||
+            appliedCalendarId == null ||
+            calendarService.getCalendars().stream().noneMatch(c -> c.getId().equals(appliedCalendarId))
```

Comment:
There is a different checkstyle issue:
'||' should be on a new line. (396:22) [OperatorWrap]
'||' should be on a new line. (397:39) [OperatorWrap]